Add conda forge badge, adjust tox configuration, fix pydocstyle (#115)
felix-hilden/sphinx-codeautolink: readme_pypi.rst

sphinx-codeautolink
===================

> |pyversions| |conda-forge| |downloads| |license| |readthedocs|

Reviewer: I don't have the PyPI versions here either, and this readme doesn't seem to be displayed on Forge (and the latest version is displayed there anyway). Therefore I'd prefer to not have this.

Installation
------------

> sphinx-codeautolink can be installed from the Python Package Index via ``pip``.
> 
> .. code:: sh
> 
>    $ pip install sphinx-codeautolink
> 
> Alternatively, for Anaconda-based distributions, using ``conda``:
> 
> .. code:: sh
> 
>    $ conda install -c conda-forge sphinx-codeautolink
> 

Reviewer: ```suggestion
sphinx-codeautolink can be installed from the following sources.

.. code:: sh

   $ pip install sphinx-codeautolink
   $ conda install -c conda-forge sphinx-codeautolink
```
